.eslintrc.js

```diff
@@ -64,10 +74,11 @@ module.exports = {
     'import/order': 0,
     'import/no-default-export': 1,
     'import/prefer-default-export': 0,
+    'react/display-name': 0,
```

Comment:
For what reason it has been added?

Reply:
The rule was enabled by config `plugin:react/recommended`, but I decided that we don't need it. If it's enabled then you're not allowed to use unnamed react components like this: `() => <Text>Some anonymous component</Text>`, and we had it in a few places.
You can enable it to see where it reports issues :smile: